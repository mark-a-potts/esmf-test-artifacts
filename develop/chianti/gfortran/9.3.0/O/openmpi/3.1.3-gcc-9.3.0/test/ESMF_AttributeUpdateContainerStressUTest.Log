build time -- 2021-04-23 11:58:54.445068
20210423 120559.445 INFO             PET0 !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
20210423 120559.445 INFO             PET0 !!! THE ESMF_LOG IS SET TO OUTPUT ALL LOG MESSAGES !!!
20210423 120559.445 INFO             PET0 !!!     THIS MAY CAUSE SLOWDOWN IN PERFORMANCE     !!!
20210423 120559.445 INFO             PET0 !!! FOR PRODUCTION RUNS, USE:                      !!!
20210423 120559.445 INFO             PET0 !!!                   ESMF_LOGKIND_Multi_On_Error  !!!
20210423 120559.445 INFO             PET0 !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
20210423 120559.445 INFO             PET0 Running with ESMF Version   : ESMF_8_2_0_beta_snapshot_05-2-g3f3d15a0c7
20210423 120559.445 INFO             PET0 ESMF library build date/time: "Apr 23 2021" "11:58:21"
20210423 120559.445 INFO             PET0 ESMF library build location : /home/mpotts/gfortran_9.3.0_openmpi_O_develop
20210423 120559.445 INFO             PET0 ESMF_COMM                   : openmpi
20210423 120559.445 INFO             PET0 ESMF_MOAB                   : enabled
20210423 120559.445 INFO             PET0 ESMF_LAPACK                 : enabled
20210423 120559.445 INFO             PET0 ESMF_NETCDF                 : enabled
20210423 120559.445 INFO             PET0 ESMF_PNETCDF                : disabled
20210423 120559.445 INFO             PET0 ESMF_PIO                    : enabled
20210423 120559.445 INFO             PET0 ESMF_YAMLCPP                : enabled
20210423 120559.445 INFO             PET0  Beginning Test, file ESMF_AttributeUpdateContainerStressUTest.F90, line 792
20210423 120559.445 INFO             PET0  NUMBER_OF_PROCESSORS           4
20210423 120559.450 ERROR            PET0 ESMCI_Info.C:668 Info::erase() Not found  - [json.exception.out_of_range.403] key 'StandardName' not found
20210423 120559.450 ERROR            PET0 ESMCI_Info.C:688 Info::erase() Not found  - Internal subroutine call returned Error
20210423 120559.450 ERROR            PET0 ESMC_InfoCDef.C:243 ESMC_InfoErase() Not found  - Internal subroutine call returned Error
20210423 120559.450 ERROR            PET0 ESMF_Info.F90:2656 ESMF_InfoRemove() Not found  - Internal subroutine call returned Error
20210423 120559.450 ERROR            PET0 src/Superstructure/AttributeAPI/interface/ESMF_Attribute.F90:45720 ESMF_AttributeRemoveAttPackField Not found  - Internal subroutine call returned Error
20210423 120559.450 ERROR            PET0 ESMCI_Info.C:668 Info::erase() Not found  - [json.exception.out_of_range.403] key 'LongName' not found
20210423 120559.450 ERROR            PET0 ESMCI_Info.C:688 Info::erase() Not found  - Internal subroutine call returned Error
20210423 120559.450 ERROR            PET0 ESMC_InfoCDef.C:243 ESMC_InfoErase() Not found  - Internal subroutine call returned Error
20210423 120559.450 ERROR            PET0 ESMF_Info.F90:2656 ESMF_InfoRemove() Not found  - Internal subroutine call returned Error
20210423 120559.450 ERROR            PET0 src/Superstructure/AttributeAPI/interface/ESMF_Attribute.F90:45720 ESMF_AttributeRemoveAttPackField Not found  - Internal subroutine call returned Error
20210423 120559.450 ERROR            PET0 ESMCI_Info.C:668 Info::erase() Not found  - [json.exception.out_of_range.403] key 'Units' not found
20210423 120559.450 ERROR            PET0 ESMCI_Info.C:688 Info::erase() Not found  - Internal subroutine call returned Error
20210423 120559.450 ERROR            PET0 ESMC_InfoCDef.C:243 ESMC_InfoErase() Not found  - Internal subroutine call returned Error
20210423 120559.450 ERROR            PET0 ESMF_Info.F90:2656 ESMF_InfoRemove() Not found  - Internal subroutine call returned Error
20210423 120559.450 ERROR            PET0 src/Superstructure/AttributeAPI/interface/ESMF_Attribute.F90:45720 ESMF_AttributeRemoveAttPackField Not found  - Internal subroutine call returned Error
20210423 120559.451 INFO             PET0  PASS  Testing for correct value on field2, ESMF_AttributeUpdateContainerStressUTest.F90, line 893
20210423 120559.451 INFO             PET0  PASS  Testing for correct value on field3, ESMF_AttributeUpdateContainerStressUTest.F90, line 905
20210423 120559.451 INFO             PET0  PASS  Testing for correct value on field6, ESMF_AttributeUpdateContainerStressUTest.F90, line 917
20210423 120559.451 ERROR            PET0 src/Superstructure/AttributeAPI/interface/ESMF_Attribute.F90:43010 ESMF_AttributeGetAttPackField() Not found  - convention/purpose not found and attpack object present
20210423 120559.451 INFO             PET0  PASS  Testing for correct value on field7, ESMF_AttributeUpdateContainerStressUTest.F90, line 932
20210423 120559.451 INFO             PET0  PASS  Testing for correct value on field8, ESMF_AttributeUpdateContainerStressUTest.F90, line 945
20210423 120559.451 INFO             PET0  Ending Test, file ESMF_AttributeUpdateContainerStressUTest.F90, line 974
20210423 120559.451 INFO             PET0 Finalizing ESMF
20210423 120559.445 INFO             PET1 !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
20210423 120559.445 INFO             PET1 !!! THE ESMF_LOG IS SET TO OUTPUT ALL LOG MESSAGES !!!
20210423 120559.445 INFO             PET1 !!!     THIS MAY CAUSE SLOWDOWN IN PERFORMANCE     !!!
20210423 120559.445 INFO             PET1 !!! FOR PRODUCTION RUNS, USE:                      !!!
20210423 120559.445 INFO             PET1 !!!                   ESMF_LOGKIND_Multi_On_Error  !!!
20210423 120559.445 INFO             PET1 !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
20210423 120559.445 INFO             PET1 Running with ESMF Version   : ESMF_8_2_0_beta_snapshot_05-2-g3f3d15a0c7
20210423 120559.445 INFO             PET1 ESMF library build date/time: "Apr 23 2021" "11:58:21"
20210423 120559.445 INFO             PET1 ESMF library build location : /home/mpotts/gfortran_9.3.0_openmpi_O_develop
20210423 120559.445 INFO             PET1 ESMF_COMM                   : openmpi
20210423 120559.445 INFO             PET1 ESMF_MOAB                   : enabled
20210423 120559.445 INFO             PET1 ESMF_LAPACK                 : enabled
20210423 120559.445 INFO             PET1 ESMF_NETCDF                 : enabled
20210423 120559.445 INFO             PET1 ESMF_PNETCDF                : disabled
20210423 120559.445 INFO             PET1 ESMF_PIO                    : enabled
20210423 120559.445 INFO             PET1 ESMF_YAMLCPP                : enabled
20210423 120559.445 INFO             PET1  Beginning Test, file ESMF_AttributeUpdateContainerStressUTest.F90, line 792
20210423 120559.445 INFO             PET1  NUMBER_OF_PROCESSORS           4
20210423 120559.450 ERROR            PET1 ESMCI_Info.C:668 Info::erase() Not found  - [json.exception.out_of_range.403] key 'StandardName' not found
20210423 120559.450 ERROR            PET1 ESMCI_Info.C:688 Info::erase() Not found  - Internal subroutine call returned Error
20210423 120559.450 ERROR            PET1 ESMC_InfoCDef.C:243 ESMC_InfoErase() Not found  - Internal subroutine call returned Error
20210423 120559.450 ERROR            PET1 ESMF_Info.F90:2656 ESMF_InfoRemove() Not found  - Internal subroutine call returned Error
20210423 120559.450 ERROR            PET1 src/Superstructure/AttributeAPI/interface/ESMF_Attribute.F90:45720 ESMF_AttributeRemoveAttPackField Not found  - Internal subroutine call returned Error
20210423 120559.450 ERROR            PET1 ESMCI_Info.C:668 Info::erase() Not found  - [json.exception.out_of_range.403] key 'LongName' not found
20210423 120559.450 ERROR            PET1 ESMCI_Info.C:688 Info::erase() Not found  - Internal subroutine call returned Error
20210423 120559.450 ERROR            PET1 ESMC_InfoCDef.C:243 ESMC_InfoErase() Not found  - Internal subroutine call returned Error
20210423 120559.450 ERROR            PET1 ESMF_Info.F90:2656 ESMF_InfoRemove() Not found  - Internal subroutine call returned Error
20210423 120559.450 ERROR            PET1 src/Superstructure/AttributeAPI/interface/ESMF_Attribute.F90:45720 ESMF_AttributeRemoveAttPackField Not found  - Internal subroutine call returned Error
20210423 120559.450 ERROR            PET1 ESMCI_Info.C:668 Info::erase() Not found  - [json.exception.out_of_range.403] key 'Units' not found
20210423 120559.450 ERROR            PET1 ESMCI_Info.C:688 Info::erase() Not found  - Internal subroutine call returned Error
20210423 120559.450 ERROR            PET1 ESMC_InfoCDef.C:243 ESMC_InfoErase() Not found  - Internal subroutine call returned Error
20210423 120559.450 ERROR            PET1 ESMF_Info.F90:2656 ESMF_InfoRemove() Not found  - Internal subroutine call returned Error
20210423 120559.450 ERROR            PET1 src/Superstructure/AttributeAPI/interface/ESMF_Attribute.F90:45720 ESMF_AttributeRemoveAttPackField Not found  - Internal subroutine call returned Error
20210423 120559.451 INFO             PET1  PASS  Testing for correct value on field2, ESMF_AttributeUpdateContainerStressUTest.F90, line 893
20210423 120559.451 INFO             PET1  PASS  Testing for correct value on field3, ESMF_AttributeUpdateContainerStressUTest.F90, line 905
20210423 120559.451 INFO             PET1  PASS  Testing for correct value on field6, ESMF_AttributeUpdateContainerStressUTest.F90, line 917
20210423 120559.451 ERROR            PET1 src/Superstructure/AttributeAPI/interface/ESMF_Attribute.F90:43010 ESMF_AttributeGetAttPackField() Not found  - convention/purpose not found and attpack object present
20210423 120559.451 INFO             PET1  PASS  Testing for correct value on field7, ESMF_AttributeUpdateContainerStressUTest.F90, line 932
20210423 120559.451 INFO             PET1  PASS  Testing for correct value on field8, ESMF_AttributeUpdateContainerStressUTest.F90, line 945
20210423 120559.451 INFO             PET1  Ending Test, file ESMF_AttributeUpdateContainerStressUTest.F90, line 974
20210423 120559.451 INFO             PET1 Finalizing ESMF
20210423 120559.445 INFO             PET2 !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
20210423 120559.445 INFO             PET2 !!! THE ESMF_LOG IS SET TO OUTPUT ALL LOG MESSAGES !!!
20210423 120559.445 INFO             PET2 !!!     THIS MAY CAUSE SLOWDOWN IN PERFORMANCE     !!!
20210423 120559.445 INFO             PET2 !!! FOR PRODUCTION RUNS, USE:                      !!!
20210423 120559.445 INFO             PET2 !!!                   ESMF_LOGKIND_Multi_On_Error  !!!
20210423 120559.445 INFO             PET2 !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
20210423 120559.445 INFO             PET2 Running with ESMF Version   : ESMF_8_2_0_beta_snapshot_05-2-g3f3d15a0c7
20210423 120559.445 INFO             PET2 ESMF library build date/time: "Apr 23 2021" "11:58:21"
20210423 120559.445 INFO             PET2 ESMF library build location : /home/mpotts/gfortran_9.3.0_openmpi_O_develop
20210423 120559.445 INFO             PET2 ESMF_COMM                   : openmpi
20210423 120559.445 INFO             PET2 ESMF_MOAB                   : enabled
20210423 120559.445 INFO             PET2 ESMF_LAPACK                 : enabled
20210423 120559.445 INFO             PET2 ESMF_NETCDF                 : enabled
20210423 120559.445 INFO             PET2 ESMF_PNETCDF                : disabled
20210423 120559.445 INFO             PET2 ESMF_PIO                    : enabled
20210423 120559.445 INFO             PET2 ESMF_YAMLCPP                : enabled
20210423 120559.445 INFO             PET2  Beginning Test, file ESMF_AttributeUpdateContainerStressUTest.F90, line 792
20210423 120559.445 INFO             PET2  NUMBER_OF_PROCESSORS           4
20210423 120559.451 INFO             PET2  PASS  Testing for correct value on field2, ESMF_AttributeUpdateContainerStressUTest.F90, line 893
20210423 120559.451 INFO             PET2  PASS  Testing for correct value on field3, ESMF_AttributeUpdateContainerStressUTest.F90, line 905
20210423 120559.451 INFO             PET2  PASS  Testing for correct value on field6, ESMF_AttributeUpdateContainerStressUTest.F90, line 917
20210423 120559.451 ERROR            PET2 src/Superstructure/AttributeAPI/interface/ESMF_Attribute.F90:43010 ESMF_AttributeGetAttPackField() Not found  - convention/purpose not found and attpack object present
20210423 120559.451 INFO             PET2  PASS  Testing for correct value on field7, ESMF_AttributeUpdateContainerStressUTest.F90, line 932
20210423 120559.451 INFO             PET2  PASS  Testing for correct value on field8, ESMF_AttributeUpdateContainerStressUTest.F90, line 945
20210423 120559.451 INFO             PET2  Ending Test, file ESMF_AttributeUpdateContainerStressUTest.F90, line 974
20210423 120559.451 INFO             PET2 Finalizing ESMF
20210423 120559.445 INFO             PET3 !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
20210423 120559.445 INFO             PET3 !!! THE ESMF_LOG IS SET TO OUTPUT ALL LOG MESSAGES !!!
20210423 120559.445 INFO             PET3 !!!     THIS MAY CAUSE SLOWDOWN IN PERFORMANCE     !!!
20210423 120559.445 INFO             PET3 !!! FOR PRODUCTION RUNS, USE:                      !!!
20210423 120559.445 INFO             PET3 !!!                   ESMF_LOGKIND_Multi_On_Error  !!!
20210423 120559.445 INFO             PET3 !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
20210423 120559.445 INFO             PET3 Running with ESMF Version   : ESMF_8_2_0_beta_snapshot_05-2-g3f3d15a0c7
20210423 120559.445 INFO             PET3 ESMF library build date/time: "Apr 23 2021" "11:58:21"
20210423 120559.445 INFO             PET3 ESMF library build location : /home/mpotts/gfortran_9.3.0_openmpi_O_develop
20210423 120559.445 INFO             PET3 ESMF_COMM                   : openmpi
20210423 120559.445 INFO             PET3 ESMF_MOAB                   : enabled
20210423 120559.445 INFO             PET3 ESMF_LAPACK                 : enabled
20210423 120559.445 INFO             PET3 ESMF_NETCDF                 : enabled
20210423 120559.445 INFO             PET3 ESMF_PNETCDF                : disabled
20210423 120559.445 INFO             PET3 ESMF_PIO                    : enabled
20210423 120559.445 INFO             PET3 ESMF_YAMLCPP                : enabled
20210423 120559.445 INFO             PET3  Beginning Test, file ESMF_AttributeUpdateContainerStressUTest.F90, line 792
20210423 120559.445 INFO             PET3  NUMBER_OF_PROCESSORS           4
20210423 120559.451 INFO             PET3  PASS  Testing for correct value on field2, ESMF_AttributeUpdateContainerStressUTest.F90, line 893
20210423 120559.451 INFO             PET3  PASS  Testing for correct value on field3, ESMF_AttributeUpdateContainerStressUTest.F90, line 905
20210423 120559.451 INFO             PET3  PASS  Testing for correct value on field6, ESMF_AttributeUpdateContainerStressUTest.F90, line 917
20210423 120559.451 ERROR            PET3 src/Superstructure/AttributeAPI/interface/ESMF_Attribute.F90:43010 ESMF_AttributeGetAttPackField() Not found  - convention/purpose not found and attpack object present
20210423 120559.451 INFO             PET3  PASS  Testing for correct value on field7, ESMF_AttributeUpdateContainerStressUTest.F90, line 932
20210423 120559.451 INFO             PET3  PASS  Testing for correct value on field8, ESMF_AttributeUpdateContainerStressUTest.F90, line 945
20210423 120559.451 INFO             PET3  Ending Test, file ESMF_AttributeUpdateContainerStressUTest.F90, line 974
20210423 120559.451 INFO             PET3 Finalizing ESMF
