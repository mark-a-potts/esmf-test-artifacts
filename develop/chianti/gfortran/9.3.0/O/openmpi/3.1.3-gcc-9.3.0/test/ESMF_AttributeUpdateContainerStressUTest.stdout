build time -- 2021-04-23 11:58:54.445068
  Beginning Test, file ESMF_AttributeUpdateContainerStressUTest.F90, line 792
  NUMBER_OF_PROCESSORS           4
  Beginning Test, file ESMF_AttributeUpdateContainerStressUTest.F90, line 792
  NUMBER_OF_PROCESSORS           4
  Beginning Test, file ESMF_AttributeUpdateContainerStressUTest.F90, line 792
  NUMBER_OF_PROCESSORS           4
  Beginning Test, file ESMF_AttributeUpdateContainerStressUTest.F90, line 792
  NUMBER_OF_PROCESSORS           4
  PASS  Testing for correct value on field2, ESMF_AttributeUpdateContainerStressUTest.F90, line 893
  PASS  Testing for correct value on field3, ESMF_AttributeUpdateContainerStressUTest.F90, line 905
  PASS  Testing for correct value on field2, ESMF_AttributeUpdateContainerStressUTest.F90, line 893
  PASS  Testing for correct value on field3, ESMF_AttributeUpdateContainerStressUTest.F90, line 905
  PASS  Testing for correct value on field6, ESMF_AttributeUpdateContainerStressUTest.F90, line 917
  PASS  Testing for correct value on field2, ESMF_AttributeUpdateContainerStressUTest.F90, line 893
  PASS  Testing for correct value on field3, ESMF_AttributeUpdateContainerStressUTest.F90, line 905
  PASS  Testing for correct value on field6, ESMF_AttributeUpdateContainerStressUTest.F90, line 917
  PASS  Testing for correct value on field2, ESMF_AttributeUpdateContainerStressUTest.F90, line 893
  PASS  Testing for correct value on field3, ESMF_AttributeUpdateContainerStressUTest.F90, line 905
  PASS  Testing for correct value on field6, ESMF_AttributeUpdateContainerStressUTest.F90, line 917
  PASS  Testing for correct value on field6, ESMF_AttributeUpdateContainerStressUTest.F90, line 917
  PASS  Testing for correct value on field7, ESMF_AttributeUpdateContainerStressUTest.F90, line 932
  PASS  Testing for correct value on field8, ESMF_AttributeUpdateContainerStressUTest.F90, line 945
  PASS  Testing for correct value on field7, ESMF_AttributeUpdateContainerStressUTest.F90, line 932
  PASS  Testing for correct value on field8, ESMF_AttributeUpdateContainerStressUTest.F90, line 945
  PASS  Testing for correct value on field7, ESMF_AttributeUpdateContainerStressUTest.F90, line 932
  PASS  Testing for correct value on field8, ESMF_AttributeUpdateContainerStressUTest.F90, line 945
  PASS  Testing for correct value on field7, ESMF_AttributeUpdateContainerStressUTest.F90, line 932
  PASS  Testing for correct value on field8, ESMF_AttributeUpdateContainerStressUTest.F90, line 945
  Ending Test, file ESMF_AttributeUpdateContainerStressUTest.F90, line 974
  Ending Test, file ESMF_AttributeUpdateContainerStressUTest.F90, line 974
  Ending Test, file ESMF_AttributeUpdateContainerStressUTest.F90, line 974
  Ending Test, file ESMF_AttributeUpdateContainerStressUTest.F90, line 974
  PET           3  Test Elapsed Time    7.9539999999999988       msec.
  PET           1  Test Elapsed Time    7.8619999999999974       msec.
  PET           0  Test Elapsed Time    7.7149999999999981       msec.
  PET           2  Test Elapsed Time    7.7959999999999994       msec.
